File: python/1043.py
e = list(map(float, input().split(' ')))
a, b, c = e[0], e[1], e[2]

if a+b > c and a+c > b and b+c > a:
    perimetro = a + b + c
    print(f"Perimetro = {perimetro:.1f}")
else:
    area = ((a + b) * c) / 2
    print(f"Area = {area:.1f}")

File: php/PROBLEMS/1043.php
<?php
    $e = fgets(STDIN);
    $e = explode(' ', $e);
    $a = $e[0];
    $b = $e[1];
    $c = $e[2];
    if ($a+$b > $c && $a+$c > $b && $b+$c > $a){
        $perimetro = $a + $b + $c;
        printf("Perimetro = %.1f\n", $perimetro);
    }
    else{
        $area = (($a + $b) * $c) / 2;
        printf("Area = %.1f\n", $area);
    }
?>
